index.js

```diff
@@ -0,0 +1,125 @@
+import * as core from '@actions/core';
+import github from '@actions/github';
+import Clubhouse from 'clubhouse-lib';
+
+export function formatMatches(matches) {
+  const values = [];
+
+  matches.forEach((match) => {
+    const regex = /\D/g;
+    const formattedMatch = match.replace(regex, '');
+    values.push(formattedMatch);
+  });
+
+  return values;
+}
+
+export function getStoryIds() {
+  const {
+    payload: { pull_request: pullRequest },
+  } = github.context;
+  const branchName = pullRequest.head.ref;
+  const branchStoryIds = branchName.match(/(ch)(\d+)/g);
+  const prTitle = pullRequest.title;
+  // Github user can enter CH story ID in either format: '[ch123]' or 'ch123':
+  const prTitleStoryIds =
+    prTitle.match(/\[(ch)(\d+)\]/g) || prTitle.match(/(ch)(\d+)/g);
```

Comment:
do we need both of these? `prTitle.match(/(ch)(\d+)/g)` might handle both cases itself?

also makes no difference but just a tip :) can match the numbers in one shot using a lookahead like:
```
/(?<=ch)\d+/g
```

Reply:
Ah yes, don't need both. haha

But also, just IN CASE (lol) if someone's username contains 'ch123', we don't wanna use that regex you're suggesting for the *branchName* I'm realizing. woops! But it's def useful for the prTitle.